### PR TITLE
Update Ollama docs.

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

File: docs/hub/ollama.md [hunk 1]
```diff
@@ -85,6 +87,14 @@ You can optionally configure a system prompt by putting it into a new file named
...
 To change sampling parameters, create a file named `params` in the repository. The file must be in JSON format. For the list of all available parameters, please refer to [this documentation](https://github.com/ollama/ollama/blob/main/docs/modelfile.md#parameter).
 
+## Run Private GGUFs from the Hugging Face Hub
+
+You can run private GGUFs from your personal account or from an associated organisation account in three simple steps:
+
+1. Copy your Ollama SSH key you can do so via: `cat ~/.ollama/id_ed25519.pub | pbcopy`
+2. Add the corresponding key to your Hugging Face account by going to [your account settings](https://huggingface.co/settings/keys) and clicking on `Add new key`.
+3. On terminal, run `huggingface-cli login` to authenticate with your Hugging Face account.
+4. That's it! You can now run private GGUFs from the Hugging Face Hub: `Ollama run hf.co/{username}/{repository}`.
```
Comment (julien-c) [inline docs/hub/ollama.md:97]:
```suggestion
4. That's it! You can now run private GGUFs from the Hugging Face Hub: `ollama run hf.co/{username}/{repository}`.
```

Comment (Vaibhavs10) [reply docs/hub/ollama.md:97]: this works w/ `ollama` too no?

Comment (Vaibhavs10) [reply docs/hub/ollama.md:97]: ah no, I'm blind, I have it the other way, fixed.

**Review by julien-c — APPROVED** [review]

File: docs/hub/ollama.md [hunk 2]
```diff
@@ -85,6 +87,13 @@ You can optionally configure a system prompt by putting it into a new file named
 
 To change sampling parameters, create a file named `params` in the repository. The file must be in JSON format. For the list of all available parameters, please refer to [this documentation](https://github.com/ollama/ollama/blob/main/docs/modelfile.md#parameter).
 
+## Run Private GGUFs from the Hugging Face Hub
+
+You can run private GGUFs from your personal account or from an associated organisation account in two simple steps:
+
+1. Copy your Ollama SSH key, you can do so via: `cat ~/.ollama/id_ed25519.pub | pbcopy`
+2. Add the corresponding key to your Hugging Face account by going to [your account settings](https://huggingface.co/settings/keys) and clicking on `Add new key`.
```
Comment (julien-c) [inline docs/hub/ollama.md:95]:
```suggestion
2. Add the corresponding key to your Hugging Face account by going to [your account settings](https://huggingface.co/settings/keys) and clicking on `Add new SSH key`.
```

**Review by julien-c — COMMENTED** [review]
woohoo!

**Pull request merged**